src/components/preview.js

```diff
@@ -0,0 +1,114 @@
+import React, { useContext } from 'react'
+import classNames from 'classnames'
+import { string, bool, oneOfType } from 'prop-types'
+
+import { partitionLine, classifyWords } from '../lib/utils'
+import { SettingsContext } from '../lib/contexts'
+
+import './preview.css'
+
+/**
+ * Overlay Line Component.
+ * Renders the various aspects of a single line.
+ * @param {string} className An optional class name to append.
+ * @param {string} gurmukhi The Gurmukhi of the line to render.
+ * @param {string} punjabiTranslation The Punjabi translation of the line to render.
+ * @param {string} englishTranslation The English translation of the line to render.
+ * @param {string} transliteration The English transliteration of the line to render.
+ */
+const Overlay = ( {
+  className,
+  gurmukhi,
+  larivaarGurbani: larivaar,
+  larivaarAssist,
+  punjabiTranslation,
+  englishTranslation,
+  spanishTranslation,
+  englishTransliteration,
+  hindiTransliteration,
+  urduTransliteration,
+} ) => {
+  const [ settings ] = useContext( SettingsContext )
+
+  larivaar = settings.larivaarGurbani
```

Comment:
you can destucture both of these out in one line, or you can delete these lines and change L58 to use `settings.larivaarGurbani && settings.larivaarAssist`